x/blob/client/cli/payforblob.go

```diff
@@ -57,10 +59,20 @@ func CmdPayForBlob() *cobra.Command {
 	}
 
 	flags.AddTxFlagsToCmd(cmd)
+	cmd.PersistentFlags().Uint8(FlagNamespaceVersion, 0, "Specify the namespace version")
 
 	return cmd
 }
 
+func getNamespace(namespaceID []byte, namespaceVersion uint8) (appns.Namespace, error) {
+	switch uint8(namespaceVersion) {
```

Comment:
this type cast is no longer necessary

```suggestion
	switch namespaceVersion {
```